tests/README.md

```diff
@@ -1,6 +1,13 @@
+# Conventions Used in This Document
+Unless otherwise noted all commands, file and directory references are relative to the *source root* directory.
+
+## Terminology
+ - modes: Collection of integration tests to be run; a mode(singular) represents an individual integration testing unit
```

Comment:
If a testing unit is a single test, then modes may encompass multiple testing units.

Reply:
I think the singular form is unnecessary to begin with so I'll just remove it.